tests/testthat/test-find_groups.R

```diff
@@ -1,13 +1,10 @@
-context("find_groups")
-
 test_that("find_groups() success case", {
-
-  options('meetupr.use_oauth' = FALSE)
+  withr::local_options(list(meetupr.use_oauth = FALSE))
```

Comment:
Doing this via withr will make the change (set on option in this case) and then restore the previous state when the test exits. Overall this makes for fewer mysteries during development, i.e. the tests should leave state as they find it.

That is NOT true currently for the API key (several tests call `set_api_key()` without storing and restoring the original value), but I gather that's going away soon anyway, so I didn't touch it.